File: semestr-8/Jezyk C#/alexander/Csharp/Archiwum/2022s/ID06TC1/Command/Command/SportsmenCommandTrening.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Command
{
    internal class SportsmenCommandTrening: SportsmenCommandComplex
    {
        private List<SportsmenCommand> commands;
        public SportsmenCommandTrening(string name, List<SportsmenCommand> commands) :base(name) { this.commands = commands; }

        public override string execute(Sportsmen sportsmen)
        {
            StringBuilder sb = new StringBuilder().Append('{');
            for(int i=0;i<commands.Count;++i)
            {
                if (i > 0) sb.Append(", ");
                sb.Append(commands[i].execute(sportsmen));
            }
            return sb.Append('}').ToString();
        }
    }
}

File: semestr-8/Programowanie obiektowe w Java/alexander/POJ/Archiwum/2021s/IZ08IO1/Command/src/SportsmenCommandTrening.java

import java.util.List;

public class SportsmenCommandTrening extends SportsmenCommandComplex
{
   private List<SportsmenCommand> list;
   public SportsmenCommandTrening(String name,List<SportsmenCommand> list)
   {
      super(name);
      this.list=list;
   }
   @Override public String execute(Sportsmen sportsmen)
   {
      StringBuilder sb=new StringBuilder();
      for(int i=0;i<list.size();++i)
      {
         if(i>0) sb.append(", ");
         sb.append(list.get(i).execute(sportsmen));
      }
      return "["+sb.toString()+"]";
   }
}
